### PR TITLE
feat(next): refactor of helm charts

### PR DESCRIPTION
**Pull request by RickyHolland**
#### What this PR does / why we need it:
PR includes the following changes.
* Removed all references to Reaction Commerce and replaced with Open Commerce.
* Removed redundant components such as Hydra, Identity, PostgreSQL.
* Updated README and other documentation as required.
* Updated Open Commerce image versions and MongoDB chart version.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: templates/NOTES.txt [hunk 1]
```diff
@@ -1,22 +1,13 @@
-Congratulations. You have just deployed Reaction Commerce!
-
-If you have any questions regarding this chart or Reaction Commerce in general, feel free to contact us at hello@slingshotlabs.io .
```
Comment (dcrdev) [inline templates/NOTES.txt:3]: @RickyHolland might want to leave this in but put hello@merchstack.com instead

File: templates/api/api-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  ## Force UTC timezone
+  TZ: "Etc/UTC"
+  NODE_ENV: "production"
+  ROOT_URL: {{ include "opencommerce.api.url" . | quote }}
+  REACTION_SHOULD_INIT_REPLICA_SET: {{ .Values.api.initReplicaSet | default "false" | quote }}
+  GRAPHQL_PLAYGROUND_ENABLED: {{ .Values.api.enableGraphQlPlayground | default "false" | quote }}
+  GRAPHQL_INTROSPECTION_ENABLED: {{ .Values.api.enableGraphQlIntrospection | default "false" | quote }}
+  STORE_URL: {{ include "opencommerce.web.url" . | quote }}
+  TOKEN_SECRET: {{ .Values.global.tokenSecret}}
+  ## required until rondlite PR is merged.
```
Comment (dcrdev) [inline templates/api/api-configmap.yaml:23]: Which PR# is this? Should add a reference to it...

**Review by dcrdev — CHANGES_REQUESTED** [review]
Couple of minor changes

Also could you give this pr a more concise name with a more fleshed out description of what you have changed?

File: templates/api/api-configmap.yaml (hunk 2 above)
Comment (RickyHolland) [reply templates/api/api-configmap.yaml:23]: I'm unable to find an outstanding or recently closed PR from rondlite so have just removed reference to it.

**Review by dcrdev — APPROVED** [review]

**Pull request merged**